governance/blogpost-process.md

```diff
@@ -0,0 +1,131 @@
+The website is part of the TAG ENV repository, blog post are located in this folder [`website/content/blog`](https://github.com/cncf/tag-env-sustainability/tree/main/website/content/blog).
+
+- [Process for creating blog post](#process-for-creating-blog-post)
+  - [Process](#process)
+    - [Writing an outline](#writing-an-outline)
+    - [Asking for reviews](#asking-for-reviews)
+    - [Incorporating Reviews](#incorporating-reviews)
+  - [Sponsorship](#sponsorship)
+  - [Review Guidelines](#review-guidelines)
+
+## Process
+
+The diagram below gives an overview of the blog post process.
+
+```mermaid
+%%{init: {'theme':'neutral'}}%%
+stateDiagram-v2
+    issue : 1. Author opens an issue\n to track the blog post
+    abstract : 2. Author writes an outline
+    reviews1 : 3. Author asks for outline reviews
+    feedback1 : 4. Author incorporates feedback
+    taglead : 5. TAG Lead approved outline and \n assigned itself to wrangle the blog
+    text : 6. Author writes the blog post
+    reviews2 : 7. Author asks for blog post reviews
+    feedback2 : 8. Author incorporates feedback
+    tagleadreview : 9. TAG Lead reviews\n and approves the blog post
+    signoff : 10. TAG Lead reaches out\n to a ToC Liaison for their final sign-off
+    merge : 11. TAG Lead merges the blog
+    
+    [*] --> issue
+    issue --> abstract
+    state Outline {
+      abstract --> reviews1
+      reviews1 --> feedback1
+      feedback1 --> reviews1
+      feedback1 --> taglead
+      taglead --> text
+    }
+    state BlogPost {
+      text --> reviews2
+      reviews2 --> feedback2
+      feedback2 --> reviews2
+      feedback2 --> tagleadreview
+      tagleadreview --> signoff
+      signoff --> merge
+    }
+    merge --> [*]
+
+```
+
+1. Open up an [issue](https://github.com/cncf/tag-env-sustainability/issues/new/choose) with the template `blog`. This issue template guides you through the process.
+2. Write an outline for your blog. Follow this section [writing an outline](#writing-an-outline) for more insights.
+3. The author asks for blog post outline reviews publicly. Follow this section [asking for reviews](#asking-for-reviews) for more insights.
+4. The author includes feedback. Follow this section [incorporating reviews](#incorporating-reviews) for more insights.
+5. As soon as the outline has been discussed feedback has been included the assigned TAG Lead will do the sign off.
+6. The author writes the blog post following the previously established outline.
+7. The author asks for blog post reviews publicly. Follow this section [asking for reviews](#asking-for-reviews) for more insights.
+8. The author includes feedback. Follow this section [incorporating reviews](#incorporating-reviews) for more insights.
+9. As soon as the blog post has been discussed feedback has been included the assigned TAG Lead will do the sign off.
+10. TAG Lead reaches out to one of the ToC Liaisons for their final sign-off
```

Comment:
I think we don't need this step, it's great to have feedback but I think that causes unnecessary work

Reply:
@mkorbi The TOC Liaison roles and responsibilities are currently in definition with this PR and the following line which defines this specifically:  https://github.com/cncf/toc/pull/1040/files#diff-0d7e3e82fc5c43ad633b1bffc1779e56f523638d0427543abf39a784a929ba68R144

Reply:
@mkorbi @TheFoxAtWork I kinda like this, because it means someone not next to the material all the time can make sure it makes sense to someone less close to the material.  I often have someone not close to the problem review my material for clarity (and these people are incredibly valuable) but this process builds it into the process.  It also makes sure we don't miss something that we may not be aware of regarding liability, like images that have copyright issues or wording that may not follow the inclusive naming conventions.